### ISSUE TITLE
Audit axe-matcher usage and increase page coverage

### ISSUE THREAD

**Issue by CGillen** [OregonDigital/OD2#880]
### Descriptive summary
Axe-matcher has been able to catch a few a11y issues before they go into the repo so we're looking at expanding its usage.

### Expected behavior
More frontend pages are being checked as accessible by axe-matcher.

A list of known pages should be produced in this ticket. That list should be marked as covered or not covered by axe-matcher after expanded coverage. Known pages that are not covered should be clear and explained why not at this time. If expected to remain uncovered and reasonable, an ADR entry provided for why not covered

### Accessibility Concerns
Increases a11y visibility & knowledge

**Comment by CGillen** [comment]
- [ ] Homepage
- [ ] Search page
- [ ] Work display page
- [ ] Collection display page
- [ ] Static pages (/help)
- [ ] Auth pages (Registration, Sign in, Password reset, Forgot password)
- [ ] Admin dashboard

**Comment by CGillen** [comment]
QA: N/A. Mostly consisted of new automated tests created. Other changes are frontend and already subject to change, pending design.
